en/concept/mixing.md

```diff
@@ -161,6 +163,19 @@ A mixer file must be named **XXXX._main_.mix** if it is responsible for the mixi
 > **Note** `XXXX` in the mixer file name is set in the airframe definition (e.g. see `set MIXER quad_w` in [airframes/10015_tbs_discovery](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/airframes/10015_tbs_discovery))
 
 
+### Loading a Custom Mixer {#loading_custom_mixer}
```

Comment:
FYI, I thought that this entry point is probably the one thing people will definitely read, so this should be part of it. We could however have as its own doc if you think that is better.

IN that case I'd probably still  have this section - but renamed "Mixer File Loading"

Reply:
Yes, this is good. Answers some comments I had above.